tests/testthat/test-helpers-simulations.R

```diff
@@ -34,3 +34,15 @@ test_that("barplot_percentages fails gracefully with bad input", {
     "Assertion on 'description' failed: Must be of type 'character', not 'double'."
   )
 })
+
+
+
+test_that("aggreagtion of additional stats works correctly", {
```

Comment:
```suggestion
test_that("aggregation of additional stats works correctly", {
```

Reply:
do we not need additional tests to make sure the new code changes work for the other designs?

Reply:
ok :)